src/components/slide-toggle/slide-toggle.ts

```diff
@@ -74,6 +76,14 @@ export class MdSlideToggle implements ControlValueAccessor {
               private _renderer: Renderer) {
   }
 
+  /** @internal */

```

Comment:
Ah right, I've heard about that in another issue, but I missed that. Thanks again for the clarification, that makes sense.